models/staging/stg_pardot.yml

```
@@ -262,10 +269,11 @@ models:
   - name: stg_pardot__opportunity_prospect
     description: Each record represents a prospect's involvement in an opportunity in Pardot.
     columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
       - name: opportunity_prospect_id
         description: Primary key for the table
```

Can add the hash details:


```suggestion
        description: Primary key for the table hashed on `source_relation`, `opportunity_id`, and prospect_id`
```